element_interface/prairieviewreader.py

```diff
@@ -0,0 +1,87 @@
+import pathlib
+import glob
+import xml.etree.ElementTree as ET
+import numpy as np
+from datetime import datetime
+
+
+
+
+def get_pv_metadata(pvtiffile):
```

Comment:
Minor point.  Please add a docstring, including the comments of no support for bidirectional, nfields=1, all planes have same dimensions, etc.  This will be helpful in interpreting how the fields are inserted in `element-calcium-imaging`:
```
self.Field.insert(
    dict(
        key,
        field_idx = plane_idx,
        px_height = pvscan_info["height_in_pixels"],
        px_width = pvscan_info["width_in_pixels"],
        um_height = pvscan_info["height_in_um"],
        um_width = pvscan_info["width_in_um"],
        )
        for plane_idx in range(pvscan_info["num_planes"])
)
```